README.md

```diff
@@ -171,6 +171,16 @@ Sometimes it is handy to have public access to Redshift clusters (for example if
 
 It is possible to integrate this VPC module with [terraform-aws-transit-gateway module](https://github.com/terraform-aws-modules/terraform-aws-transit-gateway) which handles the creation of TGW resources and VPC attachments. See [complete example there](https://github.com/terraform-aws-modules/terraform-aws-transit-gateway/tree/master/examples/complete).
 
+## Network Firewall Configuration
+If you need to use the AWS Network Firewall to provide ingress or egress restrictions that can't be accommodated easily
+by security groups, you can enable it using the following arguments:
+```hcl
+enable_networkfirewall     = true
+networkfirewall_subnets    = ["10.0.11.0/24", "10.0.12.0/24", "10.0.13.0/24"]
+networkfirewall_policy_arn = "arn:aws:network-firewall:<region>:<account_id>:firewall-policy/<policy_name>"
```

Comment:
These vars in this doc don't match the vars in the `variables.tf`

```
variable "enable_firewall" {
  description = "Should be true if you want to provision a network firewall in front of your public networks"
  type        = bool
  default     = false
}
...
variable "firewall_subnets" {
  description = "A list of firewall subnets inside the VPC"
  type        = list(string)
  default     = []
}
...
variable "firewall_policy_arn" {
  description = "The network firewall policy arn to associate with the network firewall. Needed if you are setting enable_firewall to true"
  type        = string
  default     = null
}
```

Reply:
thanks i've updated the readme for this